src/custom_modules/authorization/authMiddleware.ts

```diff
@@ -6,21 +6,23 @@ import {
   JWTPayload,
   claimIncludes,
 } from "express-oauth2-jwt-bearer";
+import {
+  authAudience,
+  authRoleClaim,
+  authIssuerBaseURL,
+  authUserIdClaim,
+} from "../../config";
 
 const authorizationRoles = require("../../enums/authorizationRoles.js");
 
+// Defaulting to "default" to enable tests to run
```

Comment:
The test suite doesn't seem to case at all what values are here, but the `auth0` throws if the values are empty.

Doesn't feel ideal, but it works. Any better ideas?